cpp/open3d/t/geometry/Image.cpp

```diff
@@ -105,7 +105,7 @@ Image Image::To(core::Dtype dtype,
             {core::Dtype::UInt16},  {core::Dtype::Int32},
             {core::Dtype::Float32}, {core::Dtype::Float64}};
 
-    [[maybe_unused]] double scale = 1.0;
+    double scale = 1.0;
```

Comment:
This variable is unused on ARM (no IPP or NPP for supporting this function), so there's a compiler error without this attribute.